c2corg_ui/static/js/card.js

```diff
@@ -26,7 +26,7 @@ app.cardDirective = function($compile, $templateCache) {
     controllerAs: 'cardCtrl',
     bindToController: true,
     scope: {
-      'doc': '=appCardDoc'
+      'doc': '<appCardDoc'
```

Comment:
Here is a good explanation: https://toddmotto.com/one-way-data-binding-in-angular-1-5/

When using a "one-way-binding" (note the difference to a "one-time-binding"), the changes inside the directive are not propagated to the outside. For objects (like in our case) it does not make a big difference though.